src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -368,59 +364,54 @@ private void addConfigParameter(String key, String value, Properties props) {
     @Override
     public void handle(Endpoint<?> endpoint, Handler<?> handler) {
         RoutingContext routingContext = (RoutingContext) endpoint.get();
-        JsonObject bodyAsJson = routingContext.getBodyAsJson();
+        JsonObject bodyAsJson = null;
+        // TODO: it seems that getBodyAsJson raises an exception when the body is empty and not null
+        try {
+            bodyAsJson = routingContext.getBodyAsJson();
+        } catch (Exception ex) {
```

Comment:
Can we use some more specific exception here instead of silently catching everything?